src/NBitcoin/Network.cs

```diff
@@ -230,6 +230,102 @@ internal static Network Register(Network network)
             return network;
         }
 
+        /// <summary>
+        /// Mines a new genesis block, to use with a new network.
+        /// Typically, 3 such genesis blocks need to be created when bootstrapping a new coin: for Main, Test and Reg networks.
+        /// </summary>
+        /// <param name="consensusFactory">
+        /// The consensus factory used to create transactions and blocks. 
+        /// Use <see cref="PosConsensusFactory"/> for proof-of-stake based networks.
+        /// </param>
+        /// <param name="pszTimestamp">
```

Comment:
We don't use Hungarians, no need to introduce them in the new code. Just timestamp would do, but if you really want to be explicit about type feel free to go with timestampString, but I would not.

Also, do we really want to college timestamp? Is it rather not Coinbase text?

Reply:
yeah it's a weird name. I was just keeping in line with bitcoin and the names we have in our current CreateGenesis methods.

Reply:
I can do coinbaseText

Reply:
in